scripts/AsciiDocBasics.gradle

```diff
@@ -26,7 +26,7 @@ if (docDir.startsWith('.')) {
     docDir = new File(projectDir, docDir).canonicalPath
 }
 if (pdfThemeDir.startsWith('.')) {
-    pdfThemeDir = new File(projectDir, pdfThemeDir).canonicalPath
+    pdfThemeDir = new File(docDir, pdfThemeDir).canonicalPath
```

Comment:
this is likely to break things and does not have to do with the `collectIncludes` (which is ok for the moment)
But it seems to be the right thing...